docs/user/precheck/check_networks.rst

```diff
@@ -0,0 +1,199 @@
+ - Busy physical networks on DST of FLAT network type
+ - Physical networks existence on DST for VLAN network type
+ - VMs spawned directly in external network
+
+
+External networks mapping
+-------------------------
+
+Checks :ref:`external networks mapping <external-networks-mapping>`. This check
+runs only if external networks IDs values are specified in the mapping file
+(``[migrate] ext_net_map`` config option contains path to this file). It
+verifies existence and validity of external networks (which sometimes referred
+as public or floating networks), specified in the map file.
+
+It simply runs ``neutron net-show <net-id>`` command and verifies that the
+networks, which are specified in the mapping file are actually exist and theirs
+IDs are valid.
+
+
+External networks overlap
+-------------------------
+
+Checks :ref:`external-networks-overlap`. Check starts only if
+``[migrate] migrate_ext_nets`` config option is set to ``True`` in the
+:ref:`primary-config-file`. This check verifies if there is any IP range
+overlapping between source and destination clouds' external networks. In
+particular, external subnets are compared. The main idea is to check if theirs
+allocation pools overlap. Only in this case migration will not start. If
+subnets have the same CIDRs, but different allocation pools, migration will
+proceed, because in this case there are no any IP conflicts. Example::
+
+    Network A (source cloud)
+    Subnet A
+    CIDR: 1.1.1.0/24
+    Allocation pool: 1.1.1.10 - 1.1.120
+
+    Network B (destination cloud)
+    Subnet B
+    CIDR: 1.1.1.0/25
+    Allocation pool: 1.1.1.5 - 1.1.1.25
+
+    Result:
+    Networks A and B overlap, cause they both have 1.1.1.10 - 1.1.1.25 IP range
+
+
+Networks segmentation IDs overlap
+---------------------------------
+
+Checks if network's segmentation ID is available on the destination. If it is
+available (not busy with any other network), segmentation ID will be kept.
+Otherwise new segmentation ID will be generated for migrated network.
+
+.. warning::
+
+    This check always passes. If segmentation ID is already used on the
+    destination cloud, ``Warning`` message will be shown during the check and
+    migration will proceed.
+
+
+Floating IPs overlap
+--------------------
+
+Checks if :ref:`Floating IPs <floating-ips>` from source cloud overlap with
+destination. If the same Floating IP address is found on the destination cloud,
+following attributes of Floating IPs will be compared:
+
+ - Tenant;
+ - External network;
+ - External network's tenant.
+
+Floating IP considered as overlapped, if any of above attributes differs,
+
+Also it checks if Floating IP is not busy (i.e. is not associated to VM on
+source and destination clouds at the same time) on both environments.
+
+
+Busy physical networks on DST of FLAT network type
+--------------------------------------------------
+
+A ``flat`` network is a network that does not provide any segmentation options.
```

Comment:
Would be good to provide a link to neutron documentation describing flat networks here